Server/LauncherClient/Owin/Controllers/GamesController.cs

```diff
@@ -30,7 +30,13 @@ public StatusMessage StartGame(int id, bool install = false)
             string baseURL = ConfigurationManager.AppSettings["BaseURL"];
             string computerKey = ConfigurationManager.AppSettings["ComputerKey"];
             string secretKey = ConfigurationManager.AppSettings["Secret"];
-            encryption = new Encryption(secretKey);
+
+            // We must decrypt the secret key using the machine key
+            //machineKeyEncryption = new Encryption("the machine key");
+            secretKey = MachineKeyEncryption.UnProtect(secretKey, "Secret");
```

Comment:
The "Secret" here should match the one you changed in "SetConfigValue" - It shouldn't work if they are different.

Reply:
Ahh, I missed that. I edited the function call to match in a145cf056cb04a7427b3191081e1ab8c745b3293 and it seems to be functioning now.